make/autoconf/util.m4

```diff
@@ -64,13 +64,17 @@ AC_DEFUN([UTIL_DEFUN_NAMED],
       )
     ])
 
-    m4_foreach([arg], [$3], [
-      m4_if(m4_bregexp(arg, [: ]), -1, m4_define([arg], m4_bpatsubst(arg, [:], [: ])))
-      m4_define(arg_name, m4_substr(arg, 0, m4_bregexp(arg, [: ])))
+    # WARNING: Underneath this comment, you will find actual demons in the foreach block below waiting to rip your very soul out and destroy everything you hold dear
+    # Proceed at the risk of your own sanity, and don't say I didn't warn you when you inevitably suffer the same torment that I went through should you decide
+    # that you need to dive into it to fix anything
+    # ~Julian
```

Comment:
🤣 While I do appreciate humor and a bit of personal touch in comments (and I fully sympathize with your description!), this feels a bit over the top. Can you tone it down a bit and make it slightly more professional (it is okay to keep a bit of the sentimentality I think, to fully make the point on how difficult it is to modify).

Reply:
Haha, sorry for letting the pent up bitterness overflow into the comments, I'll do just that. I've lost track of how much coffee and milk I've consumed while trying to get this to work...
Probably enough to shorten my lifespan by several decades at this point, if I'm being honest

Reply:
I made like two or three false starts at even getting this macro to work in the first place, after thinking "oh, that would be a good idea", trying different solutions, thinking "nah, it is impossible" to coming back to it later again with a new approach. And even with the correct approach it was basically a lot of trial and horror of permutating different m4 keywords until you got the wanted result...  So I fully understand what you have been going through. At least you can call yourself "hard-core m4 programmer" now as a result. 🎖️